app/javascript/lib/study-overview/terra-profile-warning.js

```diff
@@ -0,0 +1,10 @@
+export const profileWarning = `
+  <div class="container">
+    <p class="bg-danger">
+      The visualization above is attempting to fetch data from this study's private bucket, but it cannot issue an 
+      access token for the request as you have not completed your Terra profile. This is required in order to set your 
+      access permissions on the bucket. Please visit <a href="https://app.terra.bio" target="_blank">Terra</a> and 
+      select 'Sign in with Google' from the top left menu in order to complete your registration, and then reload this page.
```

Comment:
The help article on this would be more tailored:

```suggestion
      access permissions on the bucket. Please <a href="https://support.terra.bio/hc/en-us/articles/360028235911-How-to-register-for-a-Terra-account" target="_blank">register for Terra</a>, and then reload this page.
```

Reply:
f3422a0